app/interfaces/models.ts

```diff
@@ -36,8 +38,12 @@ export interface IUser {
   email_verified: boolean;
   auth_provider: string;
   password: string;
+  reset_password_token?: string;
+  reset_password_attempts?: number;
+  status?: UserStatuses;
   created_at: Date;
   updated_at: Date;
+  isMaxPasswordResetAttemptReached?(): boolean;
```

Comment:
I think we will need to add a guideline for variable naming as well. This is getting a little confusing.

Reply:
@abdullahtariq91 its not variable, its a function. Functions use camel casing. While the model/table keys use underscores.